features/get-all-items-breadcrumbs.feature

```diff
@@ -154,89 +100,33 @@ Scenario: Corrupt breadcrumb hierarchy (one parent-child link missing), but user
     """
 
 Scenario: Should fail when breadcrumb hierarchy is corrupt (one item missing), and user has full access to all
-  Given the database has the following table 'users':
-    | ID | sLogin | tempUser | idGroupSelf | idGroupOwned | iVersion |
-    | 1  | jdoe   | 0        | 11          | 12           | 0        |
-  And the database has the following table 'groups':
-    | ID | sName      | sTextId | iGrade | sType     | iVersion |
-    | 11 | jdoe       |         | -2     | UserAdmin | 0        |
-    | 12 | jdoe-admin |         | -2     | UserAdmin | 0        |
-    | 13 | Group B    |         | -2     | Class     | 0        |
-  And the database has the following table 'items':
-    | ID | bTeamsEditable | bNoScore | iVersion |
-    | 21 | false          | false    | 0        |
-    | 22 | false          | false    | 0        |
-    | 24 | false          | false    | 0        |
-  And the database has the following table 'items_strings':
-    | ID | idItem | idLanguage | sTitle           | iVersion |
-    | 31 | 21     | 1          | Graph: Methods   | 0        |
-    | 32 | 22     | 1          | DFS              | 0        |
-    | 33 | 24     | 1          | Reduce Graph     | 0        |
-    | 39 | 21     | 2          | Graphe: Methodes | 0        |
-  And the database has the following table 'groups_items':
+  Given the database has the following table 'groups_items':
     | ID | idGroup | idItem | sFullAccessDate | bCachedFullAccess | bCachedPartialAccess | bCachedGrayedAccess | idUserCreated | iVersion |
     | 41 | 13      | 21     | 2010-01-01      | true              | false                | false               | 0             | 0        |
     | 42 | 13      | 22     | null            | true              | false                | false               | 0             | 0        |
-    | 43 | 13      | 24     | null            | true              | false                | false               | 0             | 0        |
+    | 43 | 13      | 23     | null            | true              | false                | false               | 0             | 0        |
```

Comment:
This changes what the test is doing. It supposed to have one item missing (23) to indicate that the hierarchy is corrupt. Unless I don't understand the test case, this is incorrect.

Reply:
Hm I did notice that when I was originally reviewing and still decided to point this out, but now I don't remember why. Now when I look at it, seems to be correct 😉